proto/buf/alpha/registry/v1alpha1/search.proto

```diff
@@ -103,6 +139,9 @@ service SearchService {
   rpc SearchDraft(SearchDraftRequest) returns (SearchDraftResponse) {
     option idempotency_level = NO_SIDE_EFFECTS;
   }
+  rpc SearchContent(SearchContentRequest) returns (SearchContentResponse) {
```

Comment:
When we say "search content", it means search content within a module correct?

Maybe a more descriptive name might help (like `SearchModuleContent`), esp. if we continue expanding search then content could mean a lot of things.

Reply:
good call, updated